compiler-rt/lib/radsan/radsan_flags.inc

```diff
@@ -0,0 +1,19 @@
+//===-- radsan_flags.inc ------------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// RadSan runtime flags.
+//
+//===----------------------------------------------------------------------===//
+#ifndef RADSAN_FLAG
+# error "Define RADSAN_FLAG prior to including this file!"
+#endif
+
+// RADSAN_FLAG(Type, Name, DefaultValue, Description)
+// See COMMON_FLAG in sanitizer_flags.inc for more details.
+
+RADSAN_FLAG(const char*, error_mode, "exit", "Error mode: [continue|interactive|exit].")
```

Comment:
Super easy to define new flags in this format, very impressed with the infrastructure around this.